velocity_controllers/src/joint_group_velocity_controller.cpp

```diff
@@ -42,7 +42,7 @@ template <class T>
 void forward_command_controller::ForwardJointGroupCommandController<T>::starting(const ros::Time& time)
 {
   // Start controller with 0.0 velocities
-  commands_.resize(n_joints_, 0.0);
+  commands_buffer_.readFromRT()->assign(n_joints_, 0.0);
```

Comment:
Same thing I mention above about writing into a shared resource without synchronization primitives in place.